### DISCUSSION TITLE
Projects (Beta) Feedback - Easy to edit tickets in the table view

### DISCUSSION THREAD

**Discussion by patbeagan1** [community/community#15024] [Other Feature Feedback, Questions, & Ideas]
Hi, I wanted to lead by saying the interface is slick and the team did a great job with it. I really liked the way that you could edit issues on the same page as the table view, once they were created. 

However, once the issues were assigned to a repository, I lost the ability to edit the tickets in place. Instead of being able to edit them directly, I was sent to the issue page in that repository. It felt much clunkier, especially after having seen the more efficient UX in action. 

Is the intent to make issues in a repository editable from the Project view as well?

**Comment by Lovleyangel90** [comment]
Yeah I know 


Sent from Yahoo Mail for iPhone


On Saturday, April 16, 2022, 12:02 AM, patbeagan1 ***@***.***> wrote:




Hi, I wanted to lead by saying the interface is slick and the team did a great job with it. I really liked the way that you could edit issues on the same page as the table view, once they were created.

However, once the issues were assigned to a repository, I lost the ability to edit the tickets in place. Instead of being able to edit them directly, I was sent to the issue page in that repository. It felt much clunkier, especially after having seen the more efficient UX in action.

Is the intent to make issues in a repository editable from the Project view as well?
